src/main/java/wooteco/subway/domain/PathFinder.java

```diff
@@ -0,0 +1,51 @@
+package wooteco.subway.domain;
+
+import java.util.List;
+import org.jgrapht.GraphPath;
+import org.jgrapht.alg.shortestpath.DijkstraShortestPath;
+import org.jgrapht.graph.DefaultWeightedEdge;
+import org.jgrapht.graph.WeightedMultigraph;
+
+public class PathFinder {
+
+    final private DijkstraShortestPath<Station, DefaultWeightedEdge> dijkstraShortestPath;
+
+    public PathFinder(final List<Station> stations, final List<Section> sections) {
+        final WeightedMultigraph<Station, DefaultWeightedEdge> graph = new WeightedMultigraph(DefaultWeightedEdge.class);
```

Comment:
```suggestion
        final WeightedMultigraph<Station, DefaultWeightedEdge> graph = new WeightedMultigraph<>(DefaultWeightedEdge.class);
```
[Raw Type을 사용하면 안 되는 이유](https://mongsil1025.github.io/book/effective-java/item26/)에 대하여 알아볼까요?

Reply:
raw 타입을 사용할 경우 컴파일 타임에서 에러를 잡을 수 없어 런타임 시 예상치 못한 에러가 발생할 수 있기 때문에 raw 타입의 사용을 피해야한다는 것을 알고있었는데, 이를 무심코 지나처버렸네요..ㅠㅠ 감사합니다 범블비❤️ (이펙티브 자바 복습해야겠어요..ㅎㅎ)